scenes/game_elements/characters/player/player.tscn

```diff
@@ -1,7 +1,7 @@
 [gd_scene load_steps=34 format=3 uid="uid://iu2q66clupc6"]
 
 [ext_resource type="Script" uid="uid://bwllxup305eib" path="res://scenes/game_elements/characters/player/components/player.gd" id="1_g2els"]
-[ext_resource type="SpriteFrames" uid="uid://vwf8e1v8brdp" path="res://scenes/quests/story_quests/template/template_player_components/template_player.tres" id="2_blfj0"]
+[ext_resource type="SpriteFrames" uid="uid://vwf8e1v8brdp" path="res://scenes/quests/story_quests/you/player_components/template_player.tres" id="2_blfj0"]
```

Comment:
Similarly here, rather than changing the shared `player.tscn`, set the sprite frames on the instance of the player in your scene.

Reply:
And the same for modifying scenes and resources in the `template/` directory – these should be unmodified and the changes made to the copies in your storyquest.